docs/api/tokenization.rst

```diff
@@ -0,0 +1,4 @@
+GUFE Tokenization API
```

Comment:
Maybe something more like just `GufeTokenizable` here? This is the base class from which everything else inherits. The relevant API points aren't really about "tokenization" (indeed, all the tokenization is done in bare functions that aren't listed here)

Reply:
Tokenization -> Serialization

Reply:
Make sure there are no filename clashes.